pkg/handlers/v1/ec2.go

```diff
@@ -61,6 +61,12 @@ type ec2ConfigurationDiff struct {
 	ChangeType    string            `json:"changeType"`
 }
 
```

Comment:
the JSON structure for the object which holds the ARN in this specific message config